update faq.md, add new question (#2183)
ent/ent: doc/md/faq.md

[How to add `CHECK` constraints to table?](#how-to-add-check-constraints-to-table)  
[How to define a custom precision numeric field?](#how-to-define-a-custom-precision-numeric-field)  

> [How to configure two or more `DB` to separate read and write?](#how-to-configure-two-or-more-db-to-separate-read-and-write)

Reviewer: ```suggestion
[How to configure two or more `DB` to separate read and write?](#how-to-configure-two-or-more-db-to-separate-read-and-write)  
```

Reviewer: A line break is needed here.

}
```

#### How to change the character set and/or collation of a MySQL table?

By default for MySQL the character set `utf8mb4` is used and the collation of `utf8mb4_bin`.

> However if you'd like to change the schema's character set and/or collaction you will need to use the annotation feature.

Reviewer 2: ```suggestion
However if you'd like to change the schema's character set and/or collation you need to use an annotation.
```
